microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -118,41 +134,42 @@ By default, the **Page** tab groups resources by directory.  To display the reso
 
 1. In the **Navigator** pane (on the left), select the **Page** tab.
 
-1. Click **More options** `...` and then clear the **Group by folder** checkbox.
+1. Click **More options** `...` and then clear the **Group by folder** option.
 
+   Group by folder selected:
    ![The Group by folder option.](../media/resources-sources-page-resource-group-by-folder.msft.png)
 
-    Resources are organized by file type.  Within each file type, the resources are organized alphabetically.
+    Resources are organized by file type.  For each file type, the resources are organized alphabetically:
 
    ![The Page tab after clearing the Group by folder check mark.](../media/resources-sources-page-resources-empty-not-grouped-by-folder.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Browse resources by file type in the **Application** tool
 
-To group resources together based on their file type:
-
-1. Select the **Application** tab.  The **Application** tool opens.  By default, the **Manifest** pane usually opens first.
+Use the **Application** tool to group resources by file type:
 
+1. Select the **Application** tab.  The **Application** tool opens and by default shows the **Manifest** pane first:
+   
    ![The Application tool.](../media/resources-application-mainfest-airhorner.msft.png)
 
-1. Scroll down to the **Frames** pane.
+1. Scroll down to the **Frames** pane and then expand the folder that you're interested in:
 
    ![The Frames pane.](../media/resources-application-mainfest-airhorner-frames-expanded.msft.png)
 
-1. Expand the sections you're interested in.
-
 1. Select a resource to view it.
```

Comment:
```suggestion
1. Select a resource to view it:
```

Reply:
this change was applied, so can close this comment thread.

To check status, see:
Latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2339/files
Rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/user/v-danwesley/gs-view-page-resources/microsoft-edge/devtools-guide-chromium/resources/index.md